src/main/cpp/SwerveDrive.cpp

```diff
@@ -54,6 +56,7 @@ void SwerveDrive::updateOdometry(frc::Rotation2d robotAngle, frc::Pose2d robotPo
 **/
 void SwerveDrive::initializeAutoTraj(std::string filePath) {
     ramseteCommand_ = setupRamsete(filePath); 
+    ramseteCommand_->Initialize();

```

Comment:
how does this arrow -> work in c++?

Reply:
The arrow (->) and dot (.) operator are basically the same, they reference a public member (variable, function, etc) of an instance of a class. You use the arrow when the instance is a pointer, and a dot when the instance is not a pointer.  Here the `ramseteCommand_` object is a pointer, so the arrow is used.